src/pages/leehyeongjoon/Main/Aside/Aside.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import './Aside.scss';
+
+const Aside = () => {
+  return (
+    <div className="side-wrapper">
+      <div className="side-nickname">
+        <img
+          className="sideprofile"
+          src="/images/leehyeongjoon/profile.jpg"
+          alt=""
+        />
+        <div className="leftinformation">
+          <div className="side-id">_lhj</div>
+          <div className="threetwo">32기 이형준</div>
```

Comment:
```suggestion
        <div className="leftInformation">
          <div className="sideId">_lhj</div>
          <div className="threeTwo">32기 이형준</div>
```
className의 네이밍 컨벤션을 통일해주세요!
예시는 camelCase로 통일 했습니다.